velox/exec/PartitionedOutput.cpp

```diff
@@ -82,22 +82,23 @@ BlockingReason Destination::flush(
 }
 
 void PartitionedOutput::initializeInput(RowVectorPtr input) {
+  input_ = std::move(input);
   if (outputChannels_.empty()) {
-    input_ = std::move(input);
+    output_ = input;
```

Comment:
@oerling This is causing a crash since input has been moved into input_ a coupe lines before